src/app/pages.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
```

Comment:
I think there is a typo in the name of this service - the service which is responsible for returning places should be called PlacesService